jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/ImmutableHttpFields.java

```diff
@@ -154,6 +155,12 @@ public HttpField next()
         };
     }
 
+    @Override
+    public ListIterator<HttpField> listIterator(int index)
```

Comment:
Remove `iterator()` as its implementation is slightly incorrect (does not throw) and pretty much identical to that of `listIterator()`.